client/app/billing/payment/method/index.html

```diff
@@ -0,0 +1,125 @@
+
+    <h2 data-translate="billing_payment_method_title"></h2>
+
+    <oui-datagrid data-rows="$ctrl.paymentMethods">
+        <oui-column data-title=":: 'billing_payment_method_table_type' | translate"
+                    data-property="paymentType"
+                    data-type="options"
+                    data-type-options="$ctrl.tableFilterOptions.type"
+                    data-sortable="asc"
+                    data-filterable>
+            <span data-ng-bind="$ctrl.ovhPaymentMethodHelper.getPaymentMethodTypeText($row)"></span>
+        </oui-column>
+        <oui-column data-title=":: 'billing_payment_method_table_label' | translate"
+                    data-property="label"
+                    data-type="string"
+                    data-sortable
+                    data-searchable
+                    data-filterable>
+            <span data-ng-bind="$row.label || '-'"></span>
+        </oui-column>
+        <oui-column data-title=":: 'billing_payment_method_table_expiration_date' | translate"
+                    data-property="expirationDate">
+            <span class="d-block"
+                  data-ng-bind="$row.expirationDate ? ($row.expirationDate | date:'shortDate') : '-'">
+            </span>
+        </oui-column>
+        <oui-column data-title=":: 'billing_payment_method_table_description' | translate"
+                    data-property="description"
+                    data-type="string"
+                    data-sortable
+                    data-searchable
+                    data-filterable>
+            <span class="d-inline-block text-truncate w-25"
+                  data-ng-bind="'-'"
+                  data-ng-if="!$row.description">
+            </span>
+            <span data-ng-bind="$row.description"
+                  data-ng-if="$row.description">
+            </span>
+        </oui-column>
+        <oui-column data-title=":: 'billing_payment_method_table_default' | translate"
+                    data-property="default">
+            <span class="oui-status"
+                  data-ng-class="{
+                      'oui-status_success': $row.default,
+                      'oui-status_error': !$row.default
+                  }"
+                  data-translate="{{ $row.default ? 'common_yes' : 'common_no' }}">
+            </span>
+        </oui-column>
+        <oui-column data-title=":: 'billing_payment_method_table_state' | translate"
+                    data-property="status"
+                    data-type="options"
+                    data-type-options="$ctrl.tableFilterOptions.status"
+                    data-sortable
+                    data-filterable>
+            <span data-ng-bind="$ctrl.ovhPaymentMethodHelper.getPaymentMethodStatusText($row)"></span>
+        </oui-column>
+
+        <oui-action-menu data-align="end"
+                         data-compact>
+            <oui-action-menu-item data-text="{{ :: 'billing_payment_method_table_action_download_procedure' | translate }}"
```

Comment:
I think `text` is a deprecated attribute

Reply:
done